manual/azure-data-studio-sql-server-admin-pack/README.md

```diff
@@ -0,0 +1,23 @@
+# <img src="https://cdn.jsdelivr.net/gh/bbtsoftware/chocolatey-packages@20b4b23cc9798ce6c51f02a8d74d613f948807e9/icons/azure-data-studio-sql-server-admin-pack.png" width="48" height="48"/> [azure-data-studio-sql-server-admin-pack](https://chocolatey.org/packages/azure-data-studio-sql-server-admin-pack)
+
+Admin Pack for SQL Server is a collection of popular database administration extensions to help you manage SQL Server.
+
+## Notes
+
+When you download this pack, you will download the following extensions:
+
+> [!Note]
```

Comment:
This is not supported Markdown syntax. Just add the same sentence as done on other packages (see [here](https://github.com/bbtsoftware/chocolatey-packages/blob/master/manual/vscode-cake/vscode-cake.nuspec#L25-L26)) to the list of notes:

> * The package always installs the latest version of the extension.
>    The version of the Chocolatey package reflects not the version of the extension.